### PR TITLE
1.0.0 namedmaps

### PR DESCRIPTION
**Pull request by alrocar**
Hi Dani,

I've added the named maps APIs and some tests.

For this to work the PR at danicarrion/pyrestcli#17 has to be merged.

Anything you find "weird" please tell me...

### CODE REVIEW

**Comment by alrocar** [comment]
It works on Python 3. Also checked coding style with Atom `linter-flake8`.

I'm going to merge this into `1.0.0`

File: README.md [hunk 1]
```diff
@@ -91,74 +91,272 @@ except:
...
+# write here the ID of the dataset to retrieve
+DATASET_ID = ""
 
-im = ImportManager(auth_client)
-import_list = im.all(ids_only=False)
+dataset_manager = DatasetManager(auth_client)
+dataset = dataset_manager.get(DATASET_ID)
+
+# make the dataset PUBLIC
+dataset.privacy = PUBLIC
```
Comment (javisantana) [inline README.md]: this does use rails api?

File: README.md [hunk 2]
```diff
@@ -91,74 +91,272 @@ except:
...
+# write here the ID of the dataset to retrieve
+DATASET_ID = ""
+
+dataset_manager = DatasetManager(auth_client)
+dataset = dataset_manager.get(DATASET_ID)
+dataset.delete()
 ```
 
-You can update the attributes of an import job any time (like for checking if an import has finished):
+**Export a CARTO visualization**
```
Comment (javisantana) [inline README.md]: I don't think this is public api so it shouldn't be released or marked as beta

Comment (alrocar) [inline README.md]: I think we have added some non-public APIs. What do I do in those cases, just mark them as "beta" in the API docs?

File: README.md (hunk 1 above)
Comment (alrocar) [inline README.md]: `dataset_manager` goes through `/api/v1/viz/` (not sure if this is the rails api...)

File: README.md (hunk 2 above)
Comment (danicarrion) [inline README.md]: I'd log a warning in this case, I think

File: README.md (hunk 1 above)
Comment (javisantana) [inline README.md]: ok, this is non-public api as well so I'd use the same strategy than with .carto export

File: README.md (hunk 2 above)
Comment (alrocar) [inline README.md]: While we decide what to do, I've added warnings when using any non-public API and added warnings in the API docs as well.

From now on I continue working on branch `1.0.0`

**Pull request merged**